vigor/templates/vigor/login.html

```diff
@@ -0,0 +1,11 @@
+<form method="post">
+  {% csrf_token %}
+  Username: <input type="text" name="username" value="" size="50" />
+  Password: <input type="password" name="password" value="" size="50" />
```

Comment:
@juliancoleman A follow-up to this, see here: https://docs.djangoproject.com/ja/1.9/topics/auth/default/#all-authentication-views